libraries/provision/ansible/playbooks/prometheus.yml

```diff
@@ -10,4 +10,4 @@ scrape_configs:
     metrics_path: /_metrics
     static_configs:
       - targets:
-          - "{{sg_ip}}":4986
\ No newline at end of file
+          - 192.168.33.12:4986
```

Comment:
this looks like local ip. if ip required, get it dynamically,

Reply:
Using the  sed I am replacing the sg ip, please take a look at the start and stop prometheus method